pycbc/pnutils.py

```diff
@@ -956,3 +956,108 @@ def hybrid_meco_frequency(m1, m2, chi1, chi2, qm1=None, qm2=None):
         qm2 = 1
 
     return velocity_to_frequency(hybrid_meco_velocity(m1, m2, chi1, chi2, qm1, qm2), m1 + m2)
+
+
+def jframe_to_l0frame(mass1, mass2, f_ref, phiref=0., thetajn=0., phijl=0.,
+                      spin1_a=0., spin2_a=0.,
+                      spin1_polar=0., spin2_polar=0.,
+                      spin12_deltaphi=0.):
+    """Converts J-frame parameters into L0 frame.
+
+    Parameters
+    ----------
+    {mass1}
+    {mass2}
+    {f_ref}
```

Comment:
These and similar parts of the docstrings (parameters inside {} in inputs and outputs) will need to be written out explicitly as they're not going to be parsed like they would be in `waveform_modes.py`. For example:
```suggestion
    mass1 : float
        First component mass in solar mass units
    mass2 : float
        Second component mass in solar mass units
    f_ref: float
        Reference frequency
```